content/docs/welcome-guide/setup/setup-from-github/building-linux.md

```diff
@@ -103,10 +103,11 @@ To prepare to build the engine and projects, choose one of the following build t
 1. Use CMake to build the engine as an SDK, the same as if you installed the engine from an installer tool. The following example shows the `profile` build configuration.
 
     ```shell
-    cmake --build build/linux --target INSTALL --config profile -j <number of parallel build tasks>
+    cmake --build build/linux --target INSTALL --config <debug|profile|release> -j <number of parallel build tasks>
     ```
 
     The `-j` is a recommended build tool optimization. It tells the Ninja build tool the number of parallel build tasks that will be executed simultaneously. We recommend that the 'number of parallel build tasks' matches the number of cores available on the Linux host machine.
+    The `--config` sets configuration type. This affects compilation flags. For example, `profile` turns on `-Werror`.
```

Comment:
Mention the different build configuration types and link to the docs where the three are clearly defined. 
```suggestion
    The `--config` sets the build configuration type: `profile`, `debug`, or `release`. For setting up O3DE, `profile` is recommended. Read more on [O3DE's build configurations](/docs/user-guide/build/configure-and-build.md#generated-build-configurations).
```

If you accept this suggestion, can you also put it in the **Source engine** tab?